Box.V2/Config/Constants.cs

```diff
@@ -112,7 +112,7 @@ public static class Constants
         public const string VersionsPathString = @"{0}/versions";
         public const string CopyPathString = @"{0}/copy";
         public const string CommentsPathString = @"{0}/comments";
-        public const string ThumbnailPathString = @"{0}/thumbnail.png";
+        public const string ThumbnailPathString = @"{0}/thumbnail.{1}";
```

Comment:
Since this class and property are `public`, changing this would be backwards incompatible.  If someone had been using this constant in their own code, this change would likely break them.  You could either create a new constant here, or just do the string formatting with a string literal directly in the `BoxFilesManager` method